README.md

```diff
@@ -24,6 +26,17 @@ Use the following pattern to link revision data to Gerrit:
 * Search pattern: `(Change-Id: )#?I\w+`.
 * Nested pattern: `I\w{2,}`.
 
+
+## Compatibility Matrix
+
+
+| Git Extensions     | Gerrit Plugin       |
+|--------------------|---------------------|
+| v <= 3.5.x         | v <= 1.3.2          |
+| 3.5.x > v >= 4.0.0 | 2.0.1 >= v >= 2.0.0 |
+| v >= 4.0.1         | v >= 2.0.2          |
```

Comment:
```suggestion
| 3.5.x < v <= 4.0.0 | 2.0.0 <= v <= 2.0.1 |
| 4.0.1 <= v         | 2.0.2 <= v          |
```

Reply:
oh, thanks for letting me know)) fixed 🙂